### ISSUE TITLE
Logging verbosity for opening a connection increased in 1.3.0

### ISSUE THREAD

**Issue by azhe403** [mysql-net/MySqlConnector#956]
How to reduce logging about ping server without changing log level?

![gambar](https://user-images.githubusercontent.com/17938038/110764093-323c3580-8285-11eb-8d2a-cf6078ca7820.png)

**Comment by bgrainger** [comment]
Changing log levels _is_ the intended way to reduce logging verbosity.

FWIW, the ping message has been at `Info` level since it was introduced three years ago: https://github.com/mysql-net/MySqlConnector/commit/dc6f0586145248ed1cb06a3efe4c3e9da142a777#diff-22f5eec5ea345b8f0fd8c96f47bb2a7e59d35071c7ad2525d6f251a5ece19d5cR520

The semaphore logging is new in 1.3.0 to help diagnose the new background reset code if there are problems.

It's possible that the level of one or both of those could be reduced (to `Debug`) in the MySqlConnector code. Is there a reason that you can't set MySqlConnector logging to "WARN" or higher in your logging configuration? Are there INFO messages you need to log?

**Comment by bgrainger** [comment]
> FWIW, the ping message has been at Info level since it was introduced three years ago

But that code path may not have been executed with the old connection reset logic, so this effectively appears to be a new log statement in 1.3.0. In that case, this probably should be reduced to `Debug` to match the effective behaviour of 1.2.1 and earlier.

**Comment by bgrainger** [comment]
Logging for opening 10 connections in 1.2.1:

```
[INFO]  ConnectionPool  Pool1 creating new connection pool for ConnectionString: Server=localhost;User Id=root
[INFO]  ConnectionPool  Pool1 no pooled session available; created new Session1.1
[INFO]  ServerSession  Session1.1 connecting to IpAddress 127.0.0.1 for HostName 'localhost'
[INFO]  ServerSession  Session1.1 made connection; ServerVersion=5.7.33; ConnectionId=404; Compression=False; Attributes=True; DeprecateEof=True; Ssl=True; SessionTrack=True
```

Logging for opening 10 connections in 1.3.0:

```
[INFO]  ConnectionPool  Pool1 creating new connection pool for ConnectionString: Server=localhost;User Id=root
[INFO]  BackgroundConnectionResetHelper  Starting BackgroundConnectionResetHelper worker.
[INFO]  BackgroundConnectionResetHelper  Started BackgroundConnectionResetHelper worker.
[INFO]  BackgroundConnectionResetHelper  Waiting for semaphore.
[INFO]  BackgroundConnectionResetHelper  Waiting for semaphore.
[INFO]  ConnectionPool  Pool1 no pooled session available; created new Session1.1
[INFO]  ServerSession  Session1.1 connecting to IpAddress 127.0.0.1 for HostName 'localhost'
[INFO]  ServerSession  Session1.1 made connection; ServerVersion=5.7.33; ConnectionId=401; Compression=False; Attributes=True; DeprecateEof=True; Ssl=True; SessionTrack=True
[INFO]  ConnectionPool  Pool1 no pooled session available; created new Session1.2
[INFO]  ServerSession  Session1.2 connecting to IpAddress 127.0.0.1 for HostName 'localhost'
[INFO]  BackgroundConnectionResetHelper  Waiting for semaphore.
[INFO]  ServerSession  Session1.2 made connection; ServerVersion=5.7.33; ConnectionId=402; Compression=False; Attributes=True; DeprecateEof=True; Ssl=True; SessionTrack=True
[INFO]  BackgroundConnectionResetHelper  Waiting for semaphore.
[INFO]  ServerSession  Session1.1 successfully pinged server
[INFO]  ServerSession  Session1.2 successfully pinged server
[INFO]  ConnectionPool  Pool1 no pooled session available; created new Session1.3
[INFO]  ServerSession  Session1.3 connecting to IpAddress 127.0.0.1 for HostName 'localhost'
[INFO]  BackgroundConnectionResetHelper  Waiting for semaphore.
[INFO]  BackgroundConnectionResetHelper  Waiting for semaphore.
[INFO]  ServerSession  Session1.3 made connection; ServerVersion=5.7.33; ConnectionId=403; Compression=False; Attributes=True; DeprecateEof=True; Ssl=True; SessionTrack=True
[INFO]  ServerSession  Session1.2 successfully pinged server
[INFO]  ServerSession  Session1.1 successfully pinged server
[INFO]  ServerSession  Session1.3 successfully pinged server
[INFO]  ServerSession  Session1.2 successfully pinged server
[INFO]  ServerSession  Session1.1 successfully pinged server
[INFO]  BackgroundConnectionResetHelper  Waiting for semaphore.
[INFO]  BackgroundConnectionResetHelper  Waiting for semaphore.
```

The "ping" and "semaphore" messages are clearly an increase in verbosity since 1.2.1.

**Comment by bgrainger** [comment]
Fixed in [1.3.1](https://github.com/mysql-net/MySqlConnector/releases/tag/1.3.1).
